enhancements/network/ovn-observability-api.md

```diff
@@ -0,0 +1,290 @@
+---
+title: ovn-observability-api
+authors:
+  - npinaeva
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - dceara
+  - msherif1234
+  - jotak
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - trozet
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "None"
+creation-date: 2024-10-02
+last-updated: 2024-10-02
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/SDN-5226
+---
+
+# OVN Observability API
+
+## Summary
+
+OVN Observability is a feature that provides visibility for ovn-kubernetes-managed network by using sampling mechanism.
+That means, that network packets generate samples with user-friendly description to give more information about what and why is happening in the network.
+In openshift, this feature is integrated with the Network Observability Operator.
+
+## Motivation
+
+The purpose of this enhancement is to define an API to configure OVN observability in an openshift cluster.
+
+### User Stories
+
+- Observability: As a cluster admin, I want to enable OVN Observability integration with netobserv so that I can use network policy correlation feature.
+- Debuggability: As a cluster admin, I want to debug a networking issue using OVN Observability feature.
+
+### Goals
+
+- Design a CRD to configure OVN Observability for given user stories
+
+### Non-Goals
+
+- Discuss OVN Observability feature in general
+
+## Proposal
+
+There is a number of k8s features that can be used to configure OVN Observability right now, and some more that we expect
+to be added in the future. 
+Currently supported features are:
+- NetworkPolicy
+- (Baseline)AdminNetworkPolicy
+- EgressFirewall
+- UDN Isolation
```

Comment:
we should describe exactly what this means:

"default network pod trying to talk to the UDN pod on the default-network interface"